transports/sqs/native-integration_messagetypedetection_sqs_[6.2,).partial.md

```diff
@@ -0,0 +1,11 @@
+### Message type detection
+
+include: native-integration-messagetypedetection-intro
+
+The SQS transport can ingest any message type with no specific format and requiring no specific headers to be set. The transport applies the following loginc when receiving a message:
+
+* If the SQS message has an header named `NserviceBus.AmazonSQS.Headers`, whose value is a serialized `Dictionary<string, string>`, the message is treated as a native message. The body is deserialized into the target type [inferred from the payload](/nservicebus/messaging/message-type-detection.md). The native message body is loaded from the configured [S3 bucket](/transports/sqs/configuration-options.md#offload-large-messages-to-s3) when the `NServiceBus.AmazonSQS.Headers` attribute contains an entry with the key `S3BodyKey` and the value representing an S3 object key, including the [necessary prefix](/transports/sqs/configuration-options.md#offload-large-messages-to-s3-key-prefix) as configured by the receiving endpoint.
+* If the SQS message has an header named `MessageTypeFullName`, the message is treated as a native message and the value of the `MessageTypeFullName` will be used as the type to deserialize the message. The native message body is loaded from the configured [S3 bucket](/transports/sqs/configuration-options.md#offload-large-messages-to-s3) when the message has an `S3BodyKey` attribute with the value representing an S3 object key, including the [necessary prefix](/transports/sqs/configuration-options.md#offload-large-messages-to-s3-key-prefix) as configured by the receiving endpoint.
+* If none of the above apply, the transport tries to deserialize the message into a `TransportMessage` envelope. If it fails or there are no NServiceBus specific headers (`NServiceBus.MessageID` and `NServiceBus.EnclosedMessageTypes`), the message is treated as a native message and the message body passed as is to the processing pipeline. These is no support for loading the message body from an [S3 bucket](/transports/sqs/configuration-options.md#offload-large-messages-to-s3)
```

Comment:
Do we think adding support for loading the body from S3 in this scenario is worth it?